scripts/io_utils.py

```diff
@@ -14,3 +14,19 @@ def check_if_write(file_path, force, throw_warning=False):
             return False
     else:
         return True
+
+
+def read_csvs_with_chunksize(filename,chunksize=10000,**kwargs):
```

Comment:
It appears that with pandas >= 1.2, reading csv files with chunksize encourages context manager use (pandas-dev/pandas#38225)

I think we should use it too.

Also, I'm not familiar with the function of an empty `raise`. Will this just raise the exact error that failed the try statement?

Can you also run `black` on this file?

Reply:
Depending on how fast you need this @bethac07 - I can commit my suggestions as well. I am on vacation until late next week and won't be able to do it until then (and likely after I catch up on email)

Reply:
An empty raise will, yes, just create the same error, it's essentially just passing the error through.